docker-compose.ytt.yaml

```diff
@@ -528,6 +560,12 @@ volumes:
     driver: local
   model_configs:
     driver: local
+  storage:
```

Comment:
would be better to let docker take care of the path. You can create a new volume:
`docker volume create square-core_VOLUME_NAME`
then move data here:
`/var/lib/docker/volumes/square-core_VOLUME_NAME/_data`